global_config.py

```diff
@@ -14,7 +15,7 @@
 # URL of your Druid instance
 # Druid is a real time analytics data base. We use it to store all of our
 # analytical data (i.e. data that can be queried in AQT).
-DEFAULT_DRUID_HOST = 'https://druid.hostname.com'
+DEFAULT_DRUID_HOST = os.getenv('DEFAULT_DRUID_HOST', 'https://druid.hostname.com')
```

Comment:
Going to counter your question with a question - why is there even a 'DEFAULT_DRUID_HOST'? Shouldn't it be removed entirely? How can we assume a default host?

I would think that your configuration is never hard coded, there is no default, and you're always passing in the configuration. So I would expect `global_config.py` not to exist at all.

But yes - valid point - I've removed `https://druid.hostname.com` defaults from `docker-compose.yml` - but would prefer to remover it from global_config too.

Reply:
I'd agree with not having a fallback because that doesn't make sense and the naming of `DEFAULT_DRUID_HOST` is whack - it should be `DRUID_HOST` (which may be used elsewhere in our project and that's why default was added?). However, I wouldn't want it to have to pass it in every time. I'm not sure if something like that would better live in `global_config.py` or the docker compose, but a project is typically going to have one set Druid. So I'd like to be able to update one place  and then be able to run the web server and without passing in `DEFAULT_DRUID_HOST` every time.

Reply:
I agree in that we'd probably want to throw away `global_config.py` entirely in place for something else. Aside from that I'd want us to focus on the real goal here, to get a local docker dev environment up and running. Rather just get it working with all it's annoyances as apposed to trying to fix it all in one go (lets not open that can of worms). This can also just lead to more things to debug when there is an issue.

Reply:
I'm changing this to `DEFAULT_DRUID_HOST = os.getenv('DEFAULT_DRUID_HOST')`  There should never be hard coded configuration like this in code - that's an anti-pattern.

Reply:
Actually - couldn't stand seeing DEFAULT_DRUID_HOST - now just DRUID_HOST